pytket/docs/conf.py

```diff
@@ -55,7 +55,6 @@
 extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
-    "sphinx.ext.intersphinx",
```

Comment:
I don't think we need to remove this actually. We can probably add this back in.

Reply:
If we changes this here now we will have a merge conflict later on, is it okay to keep it like this?

Reply:
Yes should be fine.